quickwit-config/src/config.rs

```diff
@@ -478,13 +550,19 @@ impl std::fmt::Debug for QuickwitConfig {
             .field("default_index_root_uri", &self.default_index_root_uri())
             .field("indexer_config", &self.indexer_config)
             .field("searcher_config", &self.searcher_config)
-            .field("storage_config", &self.storage_config)
             .finish()
     }
 }
 
+/// Deserializes a valid [`Uri`].
+pub(super) fn _deser_valid_uri<'de, D>(deserializer: D) -> Result<Uri, D::Error>
+where D: Deserializer<'de> {
+    let uri_opt: String = Deserialize::deserialize(deserializer)?;
+    Uri::try_new(&uri_opt).map_err(D::Error::custom)
+}
+
```

Comment:
This is not being used. Should we remove it?

Reply:
Yes.